consensus/consensus.go

```diff
@@ -121,6 +124,9 @@ type Engine interface {
 	// SealHash returns the hash of a block prior to it being sealed.
 	SealHash(header *types.Header) common.Hash
 
+	// SealData signs keccak256(data)
+	SealData(data []byte) ([]byte, error)
```

Comment:
A suggestion: I think sign a `keccak256(data)` is quite dangerous, I will suggest the inputs is a bid, so that the Parlia engine can safely sign it.

Reply:
Fixed